From: SeongJae Park <sj@kernel.org>
Date: Sat, 12 Jul 2025 12:29:44 -0700
Subject: [PATCH] MAINTAINERS: rename DAMON section

DAMON section name is 'DATA ACCESS MONITOR', which implies it is only
for data access monitoring.  But DAMON is now evolved for not only
access monitoring but also access-aware system operations (DAMOS).
Rename the section to simply DAMON.  It might make it difficult to
understand what it does at a glance, but at least not spreading more
confusion.  Readers can further refer to the documentation to better
understand what really DAMON does.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 MAINTAINERS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index 4c8bbf70a3c7..ca8e3d18eedd 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -6738,7 +6738,7 @@ S:	Maintained
 W:	https://docs.dasharo.com/
 F:	drivers/platform/x86/dasharo-acpi.c
 
-DATA ACCESS MONITOR
+DAMON
 M:	SeongJae Park <sj@kernel.org>
 L:	damon@lists.linux.dev
 L:	linux-mm@kvack.org
-- 
2.39.5

